### PR TITLE
Integrate new transport layer API into Casper

### PR DESCRIPTION
**Pull request by KentShikama**
We now use the new stream endpoint to send blocks.

### Which JIRA issue does this PR relate to? If there is not a JIRA issue addressing this work, please create one now and add the link here.
Add link to corresponding JIRA issue.

https://rchain.atlassian.net/browse/RHOL-772

### CODE REVIEW

File: casper/src/test/scala/coop/rchain/casper/util/comm/TransportLayerTestImpl.scala [hunk 1]
```diff
@@ -34,15 +35,16 @@ class TransportLayerTestImpl[F[_]: Monad](
...
       handleStreamed: Blob => F[Unit]
   ): F[Unit] =
     TransportLayerTestImpl.handleQueue(dispatch, msgQueues(identity))
 
-  // FIX-ME
-  def stream(peers: Seq[PeerNode], blob: Blob): F[Unit] = ???
+  def stream(peers: Seq[PeerNode], blob: Blob): F[Unit] =
+    peers.toList
+      .traverse(peer => send(peer, protocol(blob.sender).withPacket(blob.packet)))
+      .map(_.toSeq)
```
Comment (birchmd) [inline casper/src/test/scala/coop/rchain/casper/util/comm/TransportLayerTestImpl.scala]: What is the point of this `map` since the return type is `F[Unit]`?

Or better yet, why not 
```scala
def stream(peers: Seq[PeerNode], blob: Blob): F[Unit] =
  broadcast(peers, protocol(blob.sender).withPacket(blob.packet)).void
```

File: rspace/src/main/scala/coop/rchain/rspace/LMDBOps.scala [hunk 2]
```diff
@@ -44,10 +44,6 @@ trait LMDBOps extends CloseOps {
       val ret: R = f(txn)
       txn.commit()
       ret
-    } catch {
```
Comment (birchmd) [inline rspace/src/main/scala/coop/rchain/rspace/LMDBOps.scala]: This deletion seems unrelated to the purpose of the PR.

File: comm/src/test/scala/coop/rchain/p2p/EffectsTestInstances.scala [hunk 3]
```diff
@@ -82,18 +81,12 @@ object EffectsTestInstances {
...
-          map.updated(peer, msgs)
-      }
-      broadcasted = newMap
+      requests = requests ++ peers.map(peer => Request(peer, msg))
       Seq()
     }
 
-    def stream(peers: Seq[PeerNode], blob: Blob): F[Unit] = ???
+    def stream(peers: Seq[PeerNode], blob: Blob): F[Unit] =
+      broadcast(peers, ProtocolHelper.protocol(blob.sender).withPacket(blob.packet)).as(())
```
Comment (birchmd) [inline comm/src/test/scala/coop/rchain/p2p/EffectsTestInstances.scala]: I think `.void` is clearer than `.as(())`

File: rspace/src/main/scala/coop/rchain/rspace/LMDBOps.scala (hunk 2 above)
Comment (KentShikama) [reply rspace/src/main/scala/coop/rchain/rspace/LMDBOps.scala]: It was an addition to https://github.com/rchain/rchain/pull/1647 that I decided not to commit. I think it is no longer necessary and but it definitely should go in its own PR. Thanks for the catch.

**Comment by KentShikama** [comment]
@birchmd PTAL
